packages/react-events/src/dom/Press.js

```diff
@@ -581,11 +586,12 @@ const pressResponderImpl = {
             state.activePointerId = touchEvent.identifier;
           }
 
-          // Ignore any device buttons except primary/auxillary and touch/pen contact.
+          // Ignore any device buttons except primary/secondary and touch/pen contact.
           // Additionally we ignore primary-button + ctrl-key with Macs as that
           // acts like right-click and opens the contextmenu.
           if (
-            nativeEvent.button > 1 ||
+            nativeEvent.buttons === 2 ||
```

Comment:
@trueadm When this condition included `nativeEvent.buttons != null` it caused this `Input` test to fail: https://github.com/facebook/react/blob/4279455/packages/react-events/src/dom/__tests__/Input-test.internal.js#L1038. We probably shouldn't be using `Press` in the `Input` tests or testing core logic in the responder tests.

Reply:
I agree that maybe we shouldn't be including other responder tests in other test, but we should definitely have overlapping responder tests – meaning the original test is still very much valuable.